nginx.rb

```diff
@@ -17,6 +17,7 @@
 set_default(:domain_name, "#{nginx_config_name}")
 set_default(:nginx_access_log, "#{nginx_shared_path}/log/nginx_#{nginx_config_name}_access.log")
 set_default(:nginx_error_log, "#{nginx_shared_path}/log/nginx_#{nginx_config_name}_error.log")
+set_default(:proxy_read_timeout, 75)
```

Comment:
Since this is Nginx configuration variable it's better to have name reflecting it, like `nginx_proxy_read_timeout`